book/animations.md

```diff
@@ -2292,42 +2293,43 @@ and use it to cause animated scroll in `handle_down`, by delegating scroll to
 a main thread animation.[^main-thread-scroll] You'll need to implement a new
 `ScrollAnimation` class and some logic in `run_animation_frame`. Scrolling in
 the [transform transition](examples/example13-transform-transition.html)
-example should now be smooth, as that example uses `scroll-behavior`.
+example should now be smooth, as that example uses
+`scroll-behavior`.[^scroll-linked-animations]
 
 [scroll-behavior]: https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior
 
 [^main-thread-scroll]: This will result in your browser losing threaded
 scrolling. If you've implemented the threaded animations exercise, you could
 build on that code to animate scroll on the browser thread.
 
-::: {.further}
-
-These days, many websites implement a number of *scroll-linked* animation
-effects. One common one is *parallax*. In real life, parallax is the phenomenon
-that objects further away appear to move slower than closer-in objects (due to
-the angle of light changing less quickly). For example, when riding a train,
-the trees nearby move faster across your field of view than the hills in the
-distance. The same mathematical result can be applied to web contents by way of
-the [`perspective`][perspective] CSS property. [This
-article][parallax] explains how, and [this one][csstricks-perspective]
-gives a much deeper dive into perspective in CSS generally.
+[^scroll-linked-animations]: These days, many websites implement a number
+of *scroll-linked* animation effects, such as *parallax*. In real
+life, parallax is the phenomenon that objects further away appear to move
+slower than closer-in objects (due to the angle of light changing less
+quickly). This can be achieved with the [`perspective`][perspective] CSS
+property. [This article][parallax] explains how, and
+[this one][csstricks-perspective] gives a much deeper dive into perspective
+in CSS generally.
+<br><br>
```

Comment:
Kind of ugly—I think ideally this would be two footnotes or something like that—but I'll accept it. Let the publisher complain about it (they might).

Reply:
If I were going to do two paragraphs, I'd put the second paragraph first and then add a second footnote on 3D effects in CSS and parallax.